marathon/models/events.py

```diff
@@ -28,11 +26,7 @@ def __init__(self, event_type, timestamp, **kwargs):
         self.event_type = event_type  # All events have these two attributes
         self.timestamp = timestamp
         for attribute in self.KNOWN_ATTRIBUTES:
-            try:
-                self._set(attribute, kwargs[attribute])
-            except KeyError:
-                marathon.log.warn(
-                    'Unknown event attribute processing event {}: {}'.format(event_type, attribute))
+            self._set(attribute, kwargs.get(attribute))
```

Comment:
Yes please. I'm skeptical though about how we are going to (long term) manage this event stream in python. Look at all the crazy events that might come through:
https://mesosphere.github.io/marathon/docs/event-bus.html

Depending on a list of `KNOWN_ATTRIBUTES` sounds hard to sustain long term.

I dare say what if this thing was "dumb" and just returned the raw dictionary all the time and left it up to the consumer to do what they want? I know this is departure from the rest of this library.